Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1035,6 +1056,11 @@ def genS(fragment,howMuch):
         '--pileup_input':'das:/RelValMinBiasFS_13_UP17_ForMixing/%s/GEN-SIM-RECO'%(baseDataSetRelease[15],)}
 FS_UP17_PU50_OVERLAY = merge([PUFSAVE50,Kby(100,500),steps['TTbarFS_13_UP17']] )
 
+### FastSim: template to produce signal and overlay with minbias events #PU50, 2018
+PUFSAVE50={'--pileup':'AVE_50_BX_25ns',
```

Comment:
you might need to change the name of this, since the same name is used above for 2017

Reply:
@ssekmen please fix this